src/main/java/com/epam/javalab32/maintenance_service/repository/impl/CarRepositoryImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.epam.javalab32.maintenance_service.repository.impl;
+
+import com.epam.javalab32.maintenance_service.model.Car;
+import com.epam.javalab32.maintenance_service.repository.CarRepository;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers. Check other repository classes for the same issue